packages/core/data/mm/world/snowhead_temple.yml

```diff
@@ -163,37 +167,37 @@
     "Snowhead Temple Icicles": "small_keys_sh(3) || ((has_explosives || trick_keg_explosives) && small_keys_sh(2))"
     "Snowhead Temple Center Level 2 Dual": "true"
   locations:
-    "Snowhead Temple SF Dual Switches": "can_use_lens && has_arrows && has(MASK_GREAT_FAIRY)"
+    "Snowhead Temple SF Dual Switches": "can_use_lens && (((has_arrows || can_hookshot) && has(MASK_GREAT_FAIRY)) || hookshot_anywhere)"
     "Snowhead Temple Pot Dual Switches 1": "true"
     "Snowhead Temple Pot Dual Switches 2": "true"
 "Snowhead Temple Center Level 2 Dual":
   dungeon: SH
   exits:
     "Snowhead Temple Dual Switches": "true"
-    "Snowhead Temple Map Room Upper": "goron_fast_roll || can_hookshot"
+    "Snowhead Temple Map Room Upper": "goron_fast_roll || can_hookshot || short_hook_anywhere"
     "Snowhead Temple Center Level 1": "true"
   locations:
     "Snowhead Temple Pot Central Room Scarecrow 1": "true"
     "Snowhead Temple Pot Central Room Scarecrow 2": "true"
-    "Snowhead Temple Pot Central Room Level 2 1": "true"
-    "Snowhead Temple Pot Central Room Level 2 2": "true"
```

Comment:
Where did these pots go?

Reply:
They're moved to Map Room Upper, or at least they should be.